.github/workflows/testsuite-with-db.yml

```diff
@@ -9,27 +9,22 @@ permissions:
 jobs:
   testsuite-linux:
     runs-on: ubuntu-22.04
-    continue-on-error: ${{ matrix.unstable }}
```

Comment:
We should come up with a solution to avoid having to re-add and remove such checks whenever we add a new minor.

Reply:
We can leave this and just make unstable false.